pkg/landscaper/installations/installation.go

```diff
@@ -107,6 +96,42 @@ func (i *Installation) IsImportingTarget(name string) bool {
 }
 
 // MergeConditions updates or adds the given condition to the installation's condition.
-func (i *Installation) MergeConditions(conditions ...lsv1alpha1.Condition) {
+func (i *InstallationBase) MergeConditions(conditions ...lsv1alpha1.Condition) {
 	i.Info.Status.Conditions = lsv1alpha1helper.MergeConditions(i.Info.Status.Conditions, conditions...)
 }
+
+// Installation is the internal representation of a installation
+type Installation struct {
+	*InstallationBase
```

Comment:
why is the pointer needed?
I think a pointer here makes only sense if both structs could live on their own in parallel.
Or if the installation base could be nil but that could lead to more problems.

I propose that we just embed the struct.
```suggestion
	InstallationBase
```